src/config.js

```diff
@@ -28,9 +28,10 @@ export const settings = {
 export const views = {
   ...defaultViews,
   contentTypesViews: {
+    training_class: TrainingView,
     ...defaultViews.contentTypesViews,
     person: PersonView,
-    presentation: PresentationView,
+    presentation: PresentationView
```

Comment:
The comma at the end of the line should remain, she allow to reduce commit diff and code review